packages/ts/react-signals/src/FullStackSignal.ts

```diff
@@ -0,0 +1,167 @@
+import { signal } from '@preact/signals-react';
+import type { ConnectClient, Subscription } from '@vaadin/hilla-frontend';
+import { nanoid } from 'nanoid';
+import { Signal } from './core.js';
+
+const ENDPOINT = 'SignalsHandler';
+
+type SubscriptionEvent<T> = Readonly<{
+  id: string;
+  type: StateEventType;
+  value: T;
+}>;
```

Comment:
I don't understand what's the use case for this type. 
It seems to be an unnecessary duplicate of the `StateEvent` that can be dropped.